scripts/figures/areas.R

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8-unix -*-
+library(here)
+library(tidyverse)
+
+read_tsv_helmet <- function(..., comment = "#") {
+  readr::read_tsv(..., comment = comment) %>%
+    dplyr::rename(area = X1) %>%
+    dplyr::rename_with(~ gsub("-", "_", .x, fixed = TRUE))
+}
+
+
+# Read data ---------------------------------------------------------------
+
+translations <- here::here("utilities", "areas.tsv") %>%
+  readr::read_tsv(col_types = "cc")
+
+car_density <- read_tsv_helmet(
+  here::here("data", "Tulokset", "2020", "car_density_areas.txt"),
```

Comment:
So you have these hard-coded directory paths everywhere. Is that what you mean that you should develop in another PR? Why not make the path into a variable right away?